src/mastodon/entities/notification.rs

```diff
@@ -40,6 +41,7 @@ impl Into<MegalodonEntities::notification::NotificationType> for NotificationTyp
                 MegalodonEntities::notification::NotificationType::PollExpired
             }
             NotificationType::Status => MegalodonEntities::notification::NotificationType::Status,
+            NotificationType::Update => MegalodonEntities::notification::NotificationType::Update,
```

Comment:
Please handle the same notification type in Pleroma.

Reply:
Done